### DISCUSSION TITLE
[dde-file-manager]允许第三方角标动态取消

### DISCUSSION THREAD

**Discussion by ryu-hua** [linuxdeepin/developer-center#3436] [Development Questions | 开发答疑] [answered]
最近添加的emblem插件允许三方显示自己的角标，我的插件中用代码动态控制角标的显示或取消显示，桌面图标在加载角标后无法动态取消角标。
[cacheEmblemToMap](https://github.com/linuxdeepin/dde-file-manager/blob/master/src/dde-file-manager-lib/plugins/private/pluginemblemmanagerprivate.cpp#L206)函数没有关注之前有角标现在没有角标的情况。

**Comment by GongHeng2017** [comment, marked as answer]
如果您的意思是取消显示已经设置了的角标，可以设置角标路径为""，这样能达到取消显示角标的效果。

**Reply by ryu-hua** [reply]
谢谢，我试了一下是可以的，多谢这么快的回复
